Traceback (most recent call last):
  File "C:\Users\40020507\Downloads\diagslave-3.4\win\new.py", line 43, in <module>
    from modbus_tk.defines import ModbusSlave
ImportError: cannot import name 'ModbusSlave' from 'modbus_tk.defines' (C:\Users\40020507\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\modbus_tk\defines.py)
